XenModel/Messages.resx

```diff
@@ -744,6 +744,9 @@
   <data name="ACTION_SAVE_SETTINGS" xml:space="preserve">
     <value>Save settings for '{0}'...</value>
   </data>
+  <data name="ACTION_SAVE_UPLOAD_SERVER_STATUS_REPORT" xml:space="preserve">

```

Comment:
As we are not showing the multiple action in the Events, the action description won't be visible to the user, so we don't necessarily need new resource strings for it. I think we can reuse Messages.BUGTOOL_SAVING for startDescription and Messages.COMPLETED for endDescription